File: Python_Foundation_of_Programming/32_percentage_wise_class.py
#write a program which accept percentage from user nad print class(Greater than 70:Distinction ,
#greater than 60 : First Class, Greater than 50 : Second class, Greater than 40: pass class,
#otherwise: fail)
#input: 78.9
#output:Distinction

def get_class(fper):
    pclass=''
    if fper > 70:
        pclass='Distinction'
    elif fper > 60:
        pclass='First class'
    elif fper > 50:
        pclass='Second Class'
    elif fper > 40:
        pclass='Pass'
    else:
        pclass='fail'
    return pclass

student_per =float(input("Enter Percentage:"))

print("You Got:"+get_class(student_per))

File: C_lang_Foundation_of_Programming/32_Percentagewise_class.c
//write a program which accept percentage from user and print class as (greater than 70:Distiction, greater than 60: First class,Greater than 50: second class
//,greater than 40: pass class otherwise fail.
//INPUT:78.9
//OUTPUT: Distinction

#include<stdio.h>


int main()
{
	float fpercentage=0.0f;

	printf("Prorgam to give Class depending upon percentage.\n");

	printf("Enter your percentage.\n");
	scanf("%f",&fpercentage);
	
	if(fpercentage > 75.0)
		printf("Distinction");
	else if(fpercentage > 60)
		printf("First class");
	else if(fpercentage > 50)
		printf("Second class");
	else if(fpercentage >40)
		printf("Pass Class");
	else
		printf("Fail");



	return 0;
}
